.perltidyrc

```diff
@@ -0,0 +1,18 @@
+# Workaround needed for handling non-ASCII in files.
+# # See <https://github.com/houseabsolute/perl-code-tidyall/issues/84>.
+--character-encoding=none
+--no-valign
+# TBD: openqa defines: -l=120 vs -l=160 in os-autoinst-distri-opensuse
```

Comment:
Lets stick to 160, we're not in the 80's anymore, screens are bigger and I don't expect people to do a proper review using their phones.

Reply:
Despite hearing that argument very often, the retricted line length is not about the screen width, but also about horizontal eye movement, and additionally side by side diff views are harder with longer lines.
But in my team I'm in the minority, most prefer longer lines anyway ;-)